### PR TITLE
Add release activities doc

### PR DESCRIPTION
**Pull request by dreddy-work**
Adding release activities doc to the repo.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/6891)

### CODE REVIEW

File: docs/release-activity.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+1. Merge any outstanding PRs, if necessary
+2. Ensure the HEAD of the main branch builds successfully and all tests pass
+3. Publish any unpublished API - run .\eng\ApiCompatibility\mark-shipped.cmd
+
+4. Commit and wait for a successful build
+
+5. (Milestone releases) Create "release/X.Y-qwe" branch (e.g. release/6.0-preview5) and push it into the main public trunk and the internal trunk.
+Wait for both build successfully.
+Main public trunk: [dotnet/winforms: Windows Forms is a .NET UI framework for building Windows desktop applications. (github.com)](https://github.com/dotnet/winforms/)
+Internal trunk: dotnet-winforms - Repos (azure.com)
```
Comment (RussKie) [inline docs/release-activity.md:14]:
```suggestion
* Main public trunk: https://github.com/dotnet/winforms/
* Internal trunk: <need url>
```

File: docs/release-activity.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+6. Disable loc on "release/X.Y-qwe" branch  ex: [Disable loc on release to avoid triggering locs by dreddy-work · Pull Request #6753 · dotnet/winforms (github.com)](https://github.com/dotnet/winforms/pull/6753)
```
Comment (RussKie) [inline docs/release-activity.md:15]:
```suggestion
6. Disable loc on "release/X.Y-qwe" branch (e.g.: https://github.com/dotnet/winforms/pull/6753)
```

File: docs/release-activity.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Runtime release procedures
+
+## Milestone releases (dotnet/winforms + dotnet/windowsdesktop)
+
+1. Merge any outstanding PRs, if necessary
+2. Ensure the HEAD of the main branch builds successfully and all tests pass
+3. Publish any unpublished API - run .\eng\ApiCompatibility\mark-shipped.cmd
```
Comment (RussKie) [inline docs/release-activity.md:7]:
```suggestion
3. Publish any unpublished API - run [.\eng\ApiCompatibility\mark-shipped.cmd](https://github.com/dotnet/winforms/blob/main/eng/ApiCompatibility/mark-shipped.cmd)
```

File: docs/release-activity.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+4. Commit and wait for a successful build
+
+5. (Milestone releases) Create "release/X.Y-qwe" branch (e.g. release/6.0-preview5) and push it into the main public trunk and the internal trunk.
+Wait for both build successfully.
+Main public trunk: [dotnet/winforms: Windows Forms is a .NET UI framework for building Windows desktop applications. (github.com)](https://github.com/dotnet/winforms/)
+Internal trunk: dotnet-winforms - Repos (azure.com)
+6. Disable loc on "release/X.Y-qwe" branch  ex: [Disable loc on release to avoid triggering locs by dreddy-work · Pull Request #6753 · dotnet/winforms (github.com)](https://github.com/dotnet/winforms/pull/6753)
+7. Switch back to the main branch
+8. Update the Service Fabric bot with the new release label in .\github\fabricbot.json. Refer to .\docs\infra\automation.md, if have questions.
```
Comment (RussKie) [inline docs/release-activity.md:17]: Please add links

File: docs/release-activity.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+9. Update branding.
+Here's an example: [here](https://github.com/dotnet/winforms/commit/719b2112778ffd1e7ded559ea2736b4011428117)
```
Comment (RussKie) [inline docs/release-activity.md:19]:
```suggestion
9. Update the branding (e.g.: https://github.com/dotnet/winforms/commit/719b2112778ffd1e7ded559ea2736b4011428117)
```

File: docs/release-activity.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+10. Commit and wait for a successful build.
+
+11. Create new milestone for the new release before updating following bot service.
```
Comment (RussKie) [inline docs/release-activity.md:22]:
```suggestion
11. Create new [milestone](https://github.com/dotnet/winforms/milestones) for the new release before updating following bot service.  
```

File: docs/release-activity.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+Here's an example: [here](https://github.com/dotnet/winforms/commit/719b2112778ffd1e7ded559ea2736b4011428117)
+10. Commit and wait for a successful build.
+
+11. Create new milestone for the new release before updating following bot service.  
+
+12. Move work items to this new milestone if they were on previous milestone and close the previous milestone.
+
+13. Update necessary reports with the progress
+
+14. If you are responsible for dotnet/windowsdesktop repo, make sure you change the branding there too.
```
Comment (RussKie) [inline docs/release-activity.md:28]:
```suggestion
14. If you are responsible for [dotnet/windowsdesktop](https://github.com/dotnet/windowsdesktop) repo, make sure you change the branding there too.
```

File: docs/release-activity.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+## Servicing releases (dotnet/winforms + dotnet/windowsdesktop)
+
+[NET Servicing - Overview (azure.com)](https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/545/NET-Servicing)
+
+1. Merge internal servicing branch (i.e. internal/internal/release/*) into the public branch, if necessary.  
+
+2. Update branding.
+Here's an [example](https://github.com/dotnet/winforms/commit/480bd703c5af93b9e23dc7435c6029d685be7097)
```
Comment (RussKie) [inline docs/release-activity.md:37]:
```suggestion
2. Update the branding (e.g.: https://github.com/dotnet/winforms/commit/480bd703c5af93b9e23dc7435c6029d685be7097)
```

File: docs/release-activity.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+3. Commit and wait for a successful build.
+
+4. Ensure the HEAD of the servicing branch builds successfully and all tests pass
+
+5. Update dotnet/installer with the released version.
+Here's an [example](https://github.com/dotnet/installer/commit/34ed90f436809631e254b9e15628cbcad7307629)
```
Comment (RussKie) [inline docs/release-activity.md:43]:
```suggestion
5. Update [dotnet/installer](https://github.com/dotnet/installer) with the released version (e.g.: https://github.com/dotnet/installer/commit/34ed90f436809631e254b9e15628cbcad7307629)
```

File: docs/release-activity.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+6. Create a new milestone on the GitHub
+[Milestones - dotnet/winforms (github.com)](https://github.com/dotnet/winforms/milestones?direction=asc&sort=due_date)
```
Comment (RussKie) [inline docs/release-activity.md:46]:
```suggestion
6. Create a new [milestone](https://github.com/dotnet/winforms/milestones).
```

**Review by RussKie — COMMENTED** [review]
👍  Please make sure we're consistent, and all sentences end with a full stop. Thank you.

Also I suggest moving the doc to the [wiki](https://github.com/dotnet/winforms/wiki) so we don't waste compute when we need to update it.

File: docs/release-activity.md (hunk 4 above)
Comment (dreddy-work) [reply docs/release-activity.md:17]: will do while moving this to wiki.

**Pull request merged**